modules/axhal/src/platform/loongarch64_qemu_virt/irq.rs

```diff
@@ -0,0 +1,70 @@
+use crate::irq::IrqHandler;
+use lazyinit::LazyInit;
+use loongArch64::register::{
+    ecfg::{self, LineBasedInterrupt},
+    ticlr,
+};
+
+/// The maximum number of IRQs.
+pub const MAX_IRQ_COUNT: usize = 256;
+
+/// The Extend IRQ number.
+pub const EXT_IRQ_NUM: usize = 2;
+
+/// The timer IRQ number.
+pub const TIMER_IRQ_NUM: usize = 11;
+
+static TIMER_HANDLER: LazyInit<IrqHandler> = LazyInit::new();
```

Comment:
Hardware interrupts and timer interrupts are numbered together in la64, so we can manage them uniformly with `axhal::irq::IRQ_HANDLER_TABLE`. The separate `TIMER_HANDLER` is unnecessary.